doc/api.apib

```diff
@@ -31,40 +22,43 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 
 Resources related to the Workflows registered in Styx.
 
-## Workflow [/v2/workflows/{component}/{workflow_id}]
+## Workflow [/v2/workflows/{component_id}/{workflow_id}]
 
 + Parameters
-    + component: `styx-canary` (string) - Workflow Component
-    + workflow_id: `LuigiCanary` (string) - Workflow ID
```

Comment:
Should we change the `luigi-canary-dummy:dummy` entries as well? Perhaps remove all the Luigi parts :)